packages/tooltips/src/elements/Tooltip.tsx

```diff
@@ -194,24 +157,10 @@ Tooltip.propTypes = {
   eventsEnabled: PropTypes.bool,
   id: PropTypes.string,
   content: PropTypes.node.isRequired,
-  placement: PropTypes.oneOf([
-    'auto',
-    'top',
-    'top-start',
-    'top-end',
-    'end',
-    'end-top',
-    'end-bottom',
-    'bottom',
-    'bottom-start',
-    'bottom-end',
-    'start',
-    'start-top',
-    'start-bottom'
-  ]),
+  placement: PropTypes.oneOf(PLACEMENT),
   popperModifiers: PropTypes.any,
-  size: PropTypes.oneOf(['small', 'medium', 'large', 'extra-large']),
-  type: PropTypes.oneOf(['light', 'dark']),
+  size: PropTypes.oneOf(SIZE),
+  type: PropTypes.oneOf(TYPE),
```

Comment:
Even though TypeScript helps enforce the values, all `propTypes` throughout the Garden codebase should be reusing ARRAY constants rather than maintaining separate lists.